CONTRIBUTING.md

```diff
@@ -0,0 +1,34 @@
+# Contributing
+
+The following is a set of guidelines for contributing to semeio.
+
+## Ground Rules
+
+1. We use Black code formatting
+1. We do our best to stick to pylint and flake8 code standards
+1. All code must be testable and unit tested.
+1. Commit messages should follow the format as described here https://chris.beams.io/posts/git-commit/
+
+## Pull Request Process
+
+1. Work on your own fork of the main repo
+1. Push your commits and make a draft pull request using the pull request template.
```

Comment:
Remove . for uniform formatting.

Reply:
link to pull request template?

Reply:
> link to pull request template?

This was an oversight as there is no pull request template, I deleted the text instead.